README.md

```diff
@@ -611,96 +602,69 @@ The full `options` object:
 
 For the full type definition of this, please [view our SDK API documentation](https://backstage.forgerock.com/docs/sdks/3.3/_attachments/javascript/api-reference-core/interfaces/httpclientrequestoptions.html).
 
-### Modal
+### Styling Configuration
 
-The named `modal` import provides controls of the modal component.
+The Widget can be configured for styling purposes via the JavaScript API. This allows you to choose the type of labels used or providing a logo for the modal.
 
-```js
-import { modal } from 'forgerock-login-widget/modal';
+Example:
 
-// Know when the model has been closed to run your own logic
-// The property `reason` will be either "auto", "external", or "user" (see below)
-modal.onClose(({ reason }) => {
-  /* Run anything you want */
-});
-// Know when the modal has mounted
-modal.onMount((dialogElement, formElement) => {
-  /* Run anything you want */
+```js
+const config = configuration();
+
+config.set({
+  style: {
+    checksAndRadios: 'animated', // OPTIONAL; choices are 'animated' or 'standard'
+    labels: 'floating', // OPTIONAL; choices are 'floating' or 'stacked'
+    logo: {
+      // OPTIONAL; only used with modal form factor
+      dark: 'https://example.com/img/white-logo.png', // OPTIONAL; used if theme has a dark variant
+      light: 'https://example.com/img/black-logo.png', // REQUIRED if logo property is provided; full URL
+      height: '300px', // OPTIONAL; provides additional controls to logo display
+      width: '400px', // OPTIONAL; provides additional controls to logo display
```

Comment:
When using this example as-is I saw an error stating the `height` and `width` properties should be numbers:

```
Error: samples/angular-todo/src/app/layout/header/header.component.ts:63:11 - error TS2322: Type 'string' is not assignable to type 'number'.

63           height: '300px', // OPTIONAL; provides additional controls to logo display
             ~~~~~~

  node_modules/@forgerock/login-widget/index.d.ts:2154:9
    2154         height?: number | undefined;
                 ~~~~~~
    The expected type comes from property 'height' which is declared here on type '{ height?: number; width?: number; dark?: string; light?: string; }'


Error: samples/angular-todo/src/app/layout/header/header.component.ts:64:11 - error TS2322: Type 'string' is not assignable to type 'number'.

64           width: '400px', // OPTIONAL; provides additional controls to logo display
             ~~~~~

  node_modules/@forgerock/login-widget/index.d.ts:2155:9
    2155         width?: number | undefined;
                 ~~~~~
    The expected type comes from property 'width' which is declared here on type '{ height?: number; width?: number; dark?: string; light?: string; }'
```